src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
```

Comment:
Avoid using static methods for your solution. Consider creating an instance of the Application class and moving the logic into an instance method.